torch/csrc/distributed/c10d/ddp.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <torch/csrc/utils/tensor_flatten.h>
+
+#include <c10d/ProcessGroup.hpp>
+
+#include <ATen/ATen.h>
+
+#include <cstddef>
+#include <vector>
+
+namespace c10d {
+inline void dist_broadcast_coalesced(
```

Comment:
let's keep the same coding style in c10d, distBroadcastCoalesced, and the following like broadcastOptions.